dm/master/openapi_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// this file implement all of the APIs of the DataMigration service.
+
+package master
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/pingcap/check"
+	"github.com/tikv/pd/pkg/tempurl"
+
+	"github.com/pingcap/dm/pkg/utils"
+)
+
+var openAPITestSuite = check.Suite(&openAPISuite{})
+
+type openAPISuite struct {
+	testT *testing.T
```

Comment:
where is it used?

Reply:
it will be used in next[ PR ](https://github.com/Ehco1996/dm/blob/7c01c9cacf60d882548e0bbcc2486d444c9bace2/dm/master/openapi_test.go#L811)